src/acore-wp-plugin/src/Components/UserPanel/UserController.php

```
@@ -1,175 +1,224 @@
+use ACore\Manager\ACoreServices;
+use ACore\Components\UserPanel\UserView;
+
+class UserController {
+
+    /**
+     * @var UserView
+     */
+    private $view;
+
+    public function __construct() {
+        $this->view = new UserView($this);
```

PHPStan (2.2.2)

[error] Class ACore\Components\UserPanel\UserView does not have a constructor and must be instantiated without any parameters.


(new.noConstructor)